### DISCUSSION TITLE
Couldn't access GitHub education pack all of sudden.

### DISCUSSION THREAD

**Discussion by AnupBhat30** [community/community#51015] [GitHub Education]
I got GitHub education pack on October 24 2022, today i realised GitHub copilot exists, i when on copilot page and tried to find it on GitHub education page, there it came verify yourself as student again. I tried doing it it's not working at all I'm getting rejected nearly 5 times now. We don't have a college email but GitHub is not accepting my Id card i tried clicking with different angles. What can I do to fix this.

**Comment by samus-aran** [comment]
Hi @AnupBhat30 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by AnupBhat30** [reply]
Hello,I got reverified that time but now again I lost my pack, now when i tried  reverifying it says you need to use a specific college email id, and my college hasn't issued a email id for anyone, last year they had issued only Microsoft teams ID and people used that as college email. Is there any way I can verify my education without the email id because i don't think I'll be getting it and with college ID card, receipt.
